### PR TITLE
Feat/event page

### PR DESCRIPTION
**Pull request by dhemutton**
This PR contains the addition of an events page.
Screenshot 2020-10-29 at 5 57 07 PM

### CODE REVIEW

File: src/components/Layout/NavigationBar/NavigationBar.tsx [hunk 1]
```diff
@@ -87,6 +117,21 @@ export const NavigationBar = () => {
...
+                                className="dropdown-link px-2 py-1 d-block item-center text-nowrap"
+                                smooth
+                              >
+                                {dropdownItem.label}
+                              </NavHashLink>
+                            );
+                          })}
+                        </NavDropdown>
                       ) : (
                         <NavHashLink
```
Comment (simboonlong) [inline src/components/Layout/NavigationBar/NavigationBar.tsx]: wanna try refactoring this part?

File: src/components/UI/ResourcesCard/ResourcesCard.tsx [hunk 2]
```diff
@@ -74,6 +90,24 @@ export const ResourcesCard: FunctionComponent<ResourcesCardProps> = ({ details }
...
+            <ExternalLink />
+            <span className="px-2">Event Link</span>
+          </a>
+        )}
+        {eventSlides && (
+          <a className="link" href={eventSlides} target="_blank" rel="noopener noreferrer">
+            <ExternalLink />
+            <span className="px-2">Event Slides</span>
+          </a>
+        )}
```
Comment (simboonlong) [inline src/components/UI/ResourcesCard/ResourcesCard.tsx]: Screenshot 2020-10-30 at 10 15 21 AM

File: src/components/UI/ResourcesLink/ResourcesLink.tsx [hunk 3]
```diff
@@ -13,14 +13,20 @@ export const ResourcesLink: FunctionComponent<ResourcesLinkProps> = ({ title, de
...
-                {details.description}
+                {details.title}
               </a>
             )}
+            {details.description && (
+              <>
+                <div className="description">{details.description}</div>
+                <hr />
+              </>
+            )}
```
Comment (simboonlong) [inline src/components/UI/ResourcesLink/ResourcesLink.tsx]: this is not description yeah? its the date?

File: src/pages/media.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+const sideLinks = [
+  {
+    title: "Media Centre",
+    type: "link",
+    details: [
+      {
+        title:
+          "Launch of TradeTrust announced Mr S Iswaran, Minister for Communications and Information, at the Committee of Supply Debate on 4 March 2019",
+        description: "4 Mar 2019",
```
Comment (simboonlong) [inline src/pages/media.tsx]: date?

File: src/pages/media.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  </>
+);
+
+const ResourcePageStyle = styled.div`
+  .side-links {
+    @media (min-width: ${vars.lg}) {
+      position: sticky;
+      top: 10px;
+    }
+  }
```
Comment (simboonlong) [inline src/pages/media.tsx]: for this page i think no need (content too tall already), alternatively u can make the sidebar scrollable but i'll prefer to drop sticky altogether can already

File: src/types/index.tsx [hunk 6]
```diff
@@ -26,7 +26,8 @@ export interface ResourcesLinkProps {
   type: "link" | "download";
   icon?: string;
   details: {
-    description: string;
+    title: string;
+    description?: string;
```
Comment (simboonlong) [inline src/types/index.tsx]: date?

Comment (dhemutton) [reply src/types/index.tsx]: I put description in case in the future it's used for something else 😅

File: src/pages/media.tsx (hunk 5 above)
Comment (dhemutton) [reply src/pages/media.tsx]: Ahh okie!

**Review by simboonlong — APPROVED** [review]

**Pull request merged**